src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs

```diff
@@ -2,7 +2,6 @@
 {
     public static class RouteNames
     {
-        public const string BlockedProvider_Get= "BlockedProvider_Get";
```

Comment:
Why was this removed?

Reply:
Because this is some leftover from the provider whitelist functionality which has now been removed but this stuff was remaining even though it was not used.